eZ/Bundle/EzPublishCoreBundle/Resources/views/content_fields.html.twig

```diff
@@ -410,6 +410,7 @@
 <figure {{ block( 'field_attributes' ) }}>
     {% set imageAlias = ez_image_alias( field, versionInfo, parameters.alias|default( 'original' ) ) %}
     <img src="{% if imageAlias %}{{ asset( imageAlias.uri ) }}{% else %}//:0{% endif %}"{% if imageAlias.width is defined %} width="{{ imageAlias.width }}"{% endif %}{% if imageAlias.height is defined %} height="{{ imageAlias.height }}"{% endif %} alt="{{ field.value.alternativeText }}" />
+    <img src="{% if imageAlias %}{{ asset( imageAlias.uri ) }}{% else %}//:0{% endif %}"{% if parameters.width is defined %} width="{{ parameters.width }}" {% endif %}{% if parameters.height is defined %} height="{{ parameters.height }}" {% endif %}{% if parameters.width is not defined and imageAlias.width is defined and imageAlias.width is not null %} width="{{ imageAlias.width }}"{% endif %}{% if parameters.height is not defined and imageAlias.height is defined and imageAlias.height is not null %} height="{{ imageAlias.height }}"{% endif %} alt="{{ field.value.alternativeText }}" />
```

Comment:
might be better to set a height and width variable before this line in own `set`, picking parameter first if set, then alias if not null, and last fall back being null, hence simplifying the logic on this line and improving readability overall.